.github/workflows/github-actions-PR-demo.yml

```diff
@@ -0,0 +1,46 @@
+name: GitHub Actions PR validation Demo
+on: pull_request
+jobs:
+  validate-build:
+    runs-on: ubuntu-20.04
+    steps:
+      - run: echo "Job triggered by a ${{ github.event_name }} event."
+      - run: echo "Running on a ${{ runner.os }} Github server."
+      - run: echo "Branch-ref-name is ${{ github.ref }}, repository is ${{ github.repository }}."
+      - name: Check out repository code
+        uses: actions/checkout@v2
+      - run: echo "💡 The ${{ github.repository }} repository has been cloned to the runner."
+      - run: echo "🖥️ The workflow is now ready to test your code on the runner."
+      - name: Report Maven version
+        run: |
+          mvn --version
+      - name: Run Maven test
+        run: |
+          mvn --no-transfer-progress test
+      - name: Report ${{ job.status }} status
+        run: echo "This job's status is ${{ job.status }}."
+  build-container:
+    needs: [validate-build]
+    runs-on: ubuntu-20.04
+    steps:
+    # This step will configure environment variables to be used by all steps
+    # involving AWS interaction further down
+    - name: AWS credentials configuration
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: us-east-1
+    - name: Amazon ECR login
+      id: login-ecr
+      uses: aws-actions/amazon-ecr-login@v1
+    - name: Check out repository code
+      uses: actions/checkout@v2
+    - name: Build, tag, and push image to Amazon ECR
+      env:
+        ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
+        ECR_REPOSITORY: agr_curation_prototype
```

Comment:
We have changed the repo name back to `agr_curation` so we might want to change the ECR repo name.